packages/utilities-react/src/StepFlow/index.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * Copyright IBM Corp. 2025, 2025
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+export { StepGroup } from './StepGroup';
+export { StepProvider, useStepContext } from './StepContext';
```

Comment:
```suggestion
export type { StepGroup } from './StepGroup';
export { type StepProvider, useStepContext } from './StepContext';
```

Reply:
Hmm, these aren't types so I'm thinking they should remain as they are currently exported.